SYCL/Basic/device-selectors-exception.cpp

```diff
@@ -0,0 +1,48 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN:  %t.out
```

Comment:
The simpler way to set env var:
```suggestion
// RUN:  env SYCL_DEVICE_FILTER="" %t.out
```
It doesn't require calling 'setenv()' and defining setenv() for _Win32.
E.g.: https://github.com/intel/llvm-test-suite/blob/intel/SYCL/FilterSelector/select_device.cpp#L3

Reply:
It interesting, that this test does not need any devices at all. And it is enough to run it once instead running it with CPU, then with GPU, then with ACC, CUDA, etc.
If so, it is makes sense adding it right to the compiler: https://github.com/intel/llvm/tree/sycl/sycl/test

Reply:
Thank you! Updated. Do you think `sycl/test/basic_tests` is a good place for the new test?

Reply:
Yes, "sycl/test/basic_tests" seems good enough to me.